### ISSUE TITLE
Frontend pages throw error 404 after restart

### ISSUE THREAD

**Issue by EtheriousNight** [echoCTF/echoCTF.RED#566]
**Describe the bug**
Every page except the homepage throws an "404 Page not Found"-Error after a reboot of the machine or the docker container. The Backend is working absolutly fine.

**To Reproduce**
1. Following the quick start guide from the Read.me
2. Check if the frontend is perfectly running (it should be)
3. Restart the machine or the docker container
4. Now the Error occurs

Selfmade log data:
[yii.log](https://github.com/echoCTF/echoCTF.RED/files/8480504/yii.log)

**Expected behavior**
The frontend and backend should load all sites correctly. 

**Desktop (please complete the following information):**
 - OS: Ubuntu 20 LTS
 - Browser: Firefox

**Additional context**
 My guess is that the memcache is the source of that problem, because it is the only diffrence between frontend and backend.

**Comment by proditis** [comment]
Hi @EtheriousNight,

Thanks for reporting this. Yes indeed, based on the log you provided, something isnt working well with the data being populated on memcache. Let me try and reproduce the problem and see whats causing it and i'll get back to you soon.

In the meantime can you please go to the backend Settings => Url Routes and confirm you see more than 9 records?

**Comment by EtheriousNight** [comment]
Hi @proditis,

Yes, there are 71 Entries :)

**Comment by proditis** [comment]
Ok found the culprit. Sorry it took a bit longer than expected.

So the reason of this issue is that when the database already exists no initialization scripts are loaded. This includes our own mysql-init.sql and the memcached start scripts. 

As a quick workaround (until an updated image is pushed) you can try the following commands from inside the `echoctfred_db` container (eg `docker exec -it echoctfred_db bash` and then the following commands)
```shell
/always-initdb.d/00.sh
mysql < /always-initdb.d/99_mysql-init.sql
```

This should bring all the URL rules back and the frontend will work again :crossed_fingers: (can you please confirm?)

I will leave this issue open so that the fix can be tested when ready.

**Comment by EtheriousNight** [comment]
So, I've tetstet ur update. 

The instance, that I've already had running, doesnt work anymore after the update. Thats okay, but maybe you want to take a look into this.
[console.log](https://github.com/echoCTF/echoCTF.RED/files/8486949/console.log)

The new instance is running perfectly fine and without problems.

**Comment by proditis** [comment]
Ok great, i'm glad this worked.

The problem with the collation is very odd. It seems as if the previous mariadb instance had different default charset/collations?.  I opened a new issue #568 to track this one. 

Closing this one as fixed (by #567).

Thank you for your reports :+1:
